src/Neo/SmartContract/Native/StdLib.cs

```diff
@@ -263,5 +263,13 @@ private static int StrLen([MaxLength(MaxInputLength)] string str)
 
             return count;
         }
+
+        [ContractMethod(CpuFee = 1 << 7)]
+        private static BigInteger GetRandom(ApplicationEngine engine, BigInteger maxValue)
+        {
+            if (maxValue.Sign < 0)
+                throw new ArgumentOutOfRangeException(nameof(maxValue), "Value cannot be negative.");
+            return engine.GetRandom() % (maxValue + BigInteger.One);
```

Comment:
But this is wrong. Non-uniform distribution. Try to get some statistics over `GetRandom(100)`.